gateway/manager/resharder.ts

```diff
@@ -197,6 +197,7 @@ export async function tellWorkerToPrepare(resharder: Resharder, shardId: number)
 
       if (messageData.t === "READY") {
         const payload = messageData.d as DiscordReady;
+        shard.resumeGatewayUrl = payload.resume_gateway_url;
```

Comment:
This is not necessary.
```suggestion
```

Reply:
I think its necessary for the way resharding works, a new connection can have a different resume url so we need to listen to the ready events and not ignore them